load_balanced/src/main/scala/natsconnector/NatsConfig.scala

```diff
@@ -77,6 +77,12 @@ class NatsConfig(isSource: Boolean) {
   var jsAPIPrefix: Option[String] = None // configurable
   var userName: Option[String] = None // configurable
   var userPassword: Option[String] = None // configurable
+  var sslContextFactoryClass: Option[String] = None // configurable
+  var keystorePath: Option[String] = None // configurable
+  var keystorePassword: Option[String] = None // configurable
+  var truststorePath: Option[String] = None // configurable
+  var truststorePassword: Option[String] = None // configurable
+  var tlsAlgorithm: Option[String] = None // configurable
```

Comment:
figure while I'm here, we can allow the normal ssl via parameters in addition to environment how you had it before